neutron test: nec plugin extensions location

Some of the neutron built-in testcases failed
because they can not find nec plugin extensions
based on the given the relative path.

So forcing these failure testcases to look for
these extensions using absolute path.

Signed-off-by: Vu Tran <vu.tran@windriver.com>

diff --git a/neutron/tests/unit/nec/test_nec_plugin.py b/neutron/tests/unit/nec/test_nec_plugin.py
index 2f913ca..1c3f0ce 100644
--- a/neutron/tests/unit/nec/test_nec_plugin.py
+++ b/neutron/tests/unit/nec/test_nec_plugin.py
@@ -37,7 +37,7 @@ OFC_MANAGER = 'neutron.plugins.nec.nec_plugin.ofc_manager.OFCManager'
 NOTIFIER = 'neutron.plugins.nec.nec_plugin.NECPluginV2AgentNotifierApi'
 NEC_PLUGIN_INI = """
 [DEFAULT]
-api_extensions_path = neutron/plugins/nec/extensions
+api_extensions_path = %PYTHON_SITEPACKAGES_DIR%/neutron/plugins/nec/extensions
 [OFC]
 driver = neutron.tests.unit.nec.stub_ofc_driver.StubOFCDriver
 enable_packet_filter = False
